src/components/NewApplicants/index.jsx

```diff
@@ -0,0 +1,189 @@
+import * as React from 'react';

```

Comment:
In some components you `import * as React from 'react'` and others you use `import React from 'react'` just to have the same through all the components

Reply:
Ok fixed!